api/src/pcapi/core/offerers/models.py

```diff
@@ -575,6 +575,18 @@ def current_reimbursement_point_id(self) -> int | None:
             .scalar()
         )
 
+    @current_reimbursement_point_id.expression  # type: ignore [no-redef]
+    def current_reimbursement_point_id(cls) -> str | None:  # pylint: disable=no-self-argument # type: ignore[no-redef]
+        now = datetime.utcnow()
+        return (
+            db.session.query(VenueReimbursementPointLink.reimbursementPointId)
+            .filter(
+                VenueReimbursementPointLink.venueId == cls.id,
+                VenueReimbursementPointLink.timespan.contains(now),
```

Comment:
Je pense que dans l'expression SQL on devrait plutôt écrire : 
```suggestion
                VenueReimbursementPointLink.timespan.contains(sa.func.now()),
```